scripts/beta-release.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+PROJECT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd "$PROJECT_DIR/../"
+
+npx lerna version --conventional-commits --conventional-prerelease --force-publish --include-merged-tags --preid beta --yes
```

Comment:
Have you tried this? The automagic`npx lerna version` used to cause weird issues. Skipping large numbers for no reason etc

Reply:
I'll try there first, just to make sure. But can you think of a scenario when this manifested?

I think that if we use `--include-merged-tags` it should work ok.

Reply:
The last piece of the puzzle is committing back to the repo. I don't think the changes will be committed with this.